mock/py/mockbuild/util.py

```diff
@@ -1384,16 +1386,17 @@ def include(config_file, config_opts):
     if os.path.exists(config_file):
         if config_file in config_opts['config_paths']:
             getLog().warning("Multiple inclusion of %s, skipping" % config_file)
-            return
+            return ""
 
         config_opts['config_paths'].append(config_file)
-
-        with open(config_file) as f:
-            content = f.read()
-            content = re.sub(r'include\((.*)\)', r'include(\g<1>, config_opts)', content)
-            code = compile(content, config_file, 'exec')
-        # pylint: disable=exec-used
-        exec(code)
+        content = open(config_file).read()
+        res = str_mx.findall(content)
+        if res is not None:
+            for group in res:
```

Comment:
I'd rename the `res` variable to `include_arguments`, and `group` to `include_argument`.  Or something more clear.